### DISCUSSION TITLE
Get sender location that is shared from Google Maps

### DISCUSSION THREAD

**Discussion by hajsf** [tulir/whatsmeow#146] [WhatsApp protocol Q&A]
In WhatsApp there are 2 ways usually people share location through:
1. Directly from WhatsApp
2. Sharing link from GoogleMaps

With the first "Directly from WhatsApp" I was able to read it as:
```go
case evt.Message.LocationMessage != nil:
   			Location := evt.Message.GetLocationMessage()
			fmt.Println(Location.GetDegreesLatitude())
			fmt.Println(Location.GetDegreesLongitude())
```

But did not know how to read it if the sender share it from GoogleMaps, I tried to check the difference in the message context using:
```go
		fmt.Println(evt.Message.GetMessageContextInfo())
``` 

But the output looks almost the same

**Comment by peter-lyons-kehl** [comment]
Hoping not to distract. Do I understand right that `whatsmeow` **can** get the shared location (like when I share for 1 hour or 8 hours)? I'm asking, because web.whatsapp.com does _not_ get it, and secondary Android sessions do _not_ get it either.
